README.rst

```diff
@@ -122,238 +122,11 @@ For WNS, you need both the ``WNS_PACKAGE_SECURITY_KEY`` and the ``WNS_SECRET_KEY
 
 **WP settings**
 
-- Install:
-
-.. code-block:: python
-
-	pip install pywebpush
-	pip install py-vapid  (Only for generating key)
-
-- Getting keys:
-
-	- Create file (claim.json) like this:
-
-.. code-block:: bash
-
-	{
-		"sub": "mailto: development@example.com",
-		"aud": "https://android.googleapis.com"
-	}
-
-	- Generate public and private keys:
-
-.. code-block:: bash
-
-	vapid --sign claim.json
-
-	No private_key.pem file found.
-	Do you want me to create one for you? (Y/n)Y
-	Do you want me to create one for you? (Y/n)Y
-	Generating private_key.pem
-	Generating public_key.pem
-	Include the following headers in your request:
-
-	Crypto-Key: p256ecdsa=BEFuGfKKEFp-kEBMxAIw7ng8HeH_QwnH5_h55ijKD4FRvgdJU1GVlDo8K5U5ak4cMZdQTUJlkA34llWF0xHya70
-
-	Authorization: WebPush eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJhdWQiOiJodHRwczovL2FuZHJvaWQuZ29vZ2xlYXBpcy5jb20iLCJleHAiOiIxNTA4NDkwODM2Iiwic3ViIjoibWFpbHRvOiBkZXZlbG9wbWVudEBleGFtcGxlLmNvbSJ9.r5CYMs86X3JZ4AEs76pXY5PxsnEhIFJ-0ckbibmFHZuyzfIpf1ZGIJbSI7knA4ufu7Hm8RFfEg5wWN1Yf-dR2A
-
-	- Generate client public key (applicationServerKey)
-
-.. code-block:: bash
-
-	vapid --applicationServerKey
-
-	Application Server Key = BEFuGfKKEFp-kEBMxAIw7ng8HeH_QwnH5_h55ijKD4FRvgdJU1GVlDo8K5U5ak4cMZdQTUJlkA34llWF0xHya70
-
-
-- Configure settings:
-
 - ``WP_PRIVATE_KEY``: Absolute path to your private certificate file: os.path.join(BASE_DIR, "private_key.pem")
 - ``WP_CLAIMS``: Dictionary with the same sub info like claims file: {'sub': "mailto: development@example.com"}
```

Comment:
It feels like more of an error in the docs, it's an invalid [mailto uri](https://datatracker.ietf.org/doc/html/rfc6068#page-4)
Rather than explain to people not to do that, I think it would be enough to corret the mistake in the docs
If you agree there is another place earlier in the doc that I'm aware of that has the same issue.

While we are here, this sentence doesn't make sense to me
>Dictionary with the same sub info like claims file:

I don't know what `the same sub info like claime file` is referring to
```suggestion
- ``WP_CLAIMS``: Dictionary with default value for the sub, (subject), sent to the webpush service, This would be used by the service if they needed to reach out to you, the sender. Could be a url or mailto e.g. {'sub': "mailto:development@example.com"}.
```

Reply:
There is indeed another space after `mailto:` in a configuration example earlier in the doc. And there are occurences in `tests/settings_unique.py` and `tests/settings.py`, but this is a detail.